docs/source/howto/installation.rst

```diff
@@ -314,7 +314,45 @@ Tuning performance
 
 AiiDA supports running hundreds of thousands of calculations and graphs with millions of nodes.
 However, optimal performance at that scale might require some tweaks to the AiiDA configuration to balance the CPU and disk load.
-Here are a few general tips that might improve the AiiDA performance:
+
+Here are a few tips for tuning AiiDA performance:
+
+
+    .. dropdown:: Increase the number of daemon workers
+
+        By default, the AiiDA daemon only uses a single worker, i.e. a single operating system process.
+        If ``verdi daemon status`` shows the daemon worker constantly at high CPU usage, you can use ``verdi daemon incr X`` to add ``X`` parallel daemon workers.
+
+        Keep in mind that other processes need to run on your computer (e.g. rabbitmq, the PostgreSQL database, ...), i.e. you likely want to stop increasing the number of workers *before* you reach the number of cores of your CPU.
+
+        To make the change permanent, set
+        ::
+
+            verdi config set daemon.default_workers 5
+
+    .. dropdown:: Increase the number of daemon worker slots
+
+
+        By default, each daemon worker accepts only 200 tasks at a time.
+        If ``verdi daemon status`` constantly warns about a high percentage of the available daemon worker slots being used, you can increase the number of tasks handled by each daemon worker (thus increasing the workload per worker).
```

Comment:
I think this is in `verdi process list` and not `verdi daemon status`.
```suggestion
        If ``verdi process list`` warns about a high percentage of the available daemon worker slots being used, you can increase the number of tasks handled by each daemon worker (thus increasing the workload per worker).
```

Reply:
Maybe we should say this is an alternative to running more daemon workers if you don't have enough CPUs, but that of  course each machine will have a max at some point and we cannot increase infinitely. It is difficult if not impossible to give settings that work for everyone, because some people might try to run on a small 2 CPU laptop and others have a dedicated 64 CPU workstation or server. I think this is important to convey somehow

Reply:
Yes, it also depends on the Processes you are running (e.g. if they involve significant computation or queries to the DB, ...).
I've tried to incorporate this now